src/conf_mode/system_option.py

```diff
@@ -155,6 +156,13 @@ def apply(options):
         time_format = time_format_to_locale.get(options['time_format'])
         cmd(f'localectl set-locale LC_TIME={time_format}')
 
+    # Enable/disable dynamic debugging for Wireguard module
+    if 'debug_wireguard' in options.get('kernel', {}):
+        check_kmod('wireguard')
+        cmd('echo "module wireguard +p" | sudo tee /sys/kernel/debug/dynamic_debug/control')
```

Comment:
There should be probably better way to write changes to the file.
Echo good for bash.

Reply:
We generally use write_file() to interact with sysfs